example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -230,7 +231,7 @@ auto example(TAccTag const&) -> int
     alpaka::memcpy(queue, bufAccS, bufHostS);
     RunTimestepKernelSingle runTimestepKernelSingle;
 
-    auto const& bundeledKernelRuntimeStep = alpaka::KernelBundle(
+    auto const& kernelBundleRuntimeStep = alpaka::KernelBundle(
```

Comment:
```suggestion
    auto const kernelBundleRuntimeStep = alpaka::KernelBundle(
```
?

Reply:
We have it already in the mainline. It is valid because `const&` is extending the live time of the temporary. 
I am no fan of using it but since @mehmetyusufoglu did it already everywhere I kept it.

Reply:
I see... OK, we can fix it later :-)